CHANGELOG.txt

```diff
@@ -7,6 +7,8 @@ master:
    * UTCDateTime objects now issue depreciation warnings when setting any
      attributes outside of init, or comparing UTCDateTime objects with
      different precisions (see #2077).
+   * UTCDateTime objects can now except hour, minute, and second values
```

Comment:
s/except/accept/ (though technically the former is valid in this case, I'm not sure it's what you meant)

Reply:
right, this was a typo, thanks.